### DISCUSSION TITLE
SSL between NGING Proxy Manager and Vaultwarden

### DISCUSSION THREAD

**Discussion by DaveSH12** [dani-garcia/vaultwarden#4872] [Q&A] [answered]
Hey Community,
I'm using nginx Proxy Manager and I want to secure the connection between NGINGX Proxy Manager and Vaultwarden. I know how to setup the SSL Certs with RocketTLS, withour NGINX. But how can I use SSL with NGINX? I know about the Force SSL Option in NGINX Proxy Manager. But we want to be safe, that all connections are always using SSL. 

Kind regards,
David

**Comment by senna1992** [comment]
I have set it to force ssl and and protocol https. Maybe that's what you are looking for?

**Reply by DaveSH12** [reply]
Yes, but when I set protocol to https my Vaultwarden isn't reachable. How can I configure my Vaultwarden using SSL?

**Reply by DaveSH12** [reply, marked as answer]
Okay, I found it. Activating Rocket TLS in Config without configuring the port was the solution.
Thank you.
